pandas/core/series.py

```diff
@@ -2764,7 +2764,7 @@ def combine_first(self, other) -> "Series":
 
         Parameters
         ----------
-        other : Series
+        other : Series, or object coercible into Series
```

Comment:
It looks like you're editing the docstring of combine_first, can you modify the one for update instead? @WillAyd do you think it'd be good to add an example to the docstring for this behavior?